staging/src/k8s.io/kubectl/pkg/cmd/run/run.go

```diff
@@ -311,10 +302,7 @@ func (o *RunOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []string) e
 	}
 
 	generators := generateversioned.GeneratorFn("run")
-	generator, found := generators[generateversioned.RunPodV1GeneratorName]
```

Comment:
I know we are dropping the generator flag, but I'm wondering if we still shouldn't be checking the existence of this on the map. Otherwise, on the next line (`names := generator.ParamNames()`) we might be calling a nil value and get a panic, if someone removes this generator :)